src/everest/config/validation_utils.py

```diff
@@ -139,13 +139,13 @@ def unique_items(items: Sequence[T]) -> Sequence[T]:
     return items
 
 
-def valid_range(range_value: tuple[float, float]):
+def valid_range(range_value: tuple[float, float]) -> tuple[float, float]:
     if range_value[0] >= range_value[1]:
         raise ValueError("scaled_range must be a valid range [a, b], where a < b.")
     return range_value
 
 
-def check_path_valid(path: str):
+def check_path_valid(path: str | None) -> None:
```

Comment:
Why allow None  if we have code that raises an error if we have anything else than a string? 
``` python
    if not isinstance(path, str):
        raise ValueError("str type expected")
```

Reply:
I guess in this case a better type for paths is Any